crates/wasmtime/src/runtime/code_memory.rs

```diff
@@ -182,6 +186,16 @@ impl CodeMemory {
         #[cfg(not(has_host_compiler_backend))]
         let _ = &mut unwind;
 
+        // Ensure that the exception table is well-formed. This parser
+        // construction is cheap: it reads the header and validates
+        // ranges but nothing else. We do this only in debug-assertion
+        // builds because we otherwise require for safety that the
+        // compiled artifact is as-produced-by this version of
+        // Wasmtime, and we should always produce a correct exception
+        // table (i.e., we are not expecting untrusted data here).
+        #[cfg(debug_assertions)]
```

Comment:
Similar to above, could this use `if cfg!(debug_assertions) { ... }` to avoid needing a conditional import on `ExceptionTable`? (this'll show up in CI for release builds I think which aren't enabled by default on PRs)